test/unit/processing/siti-agri/transform-invoice-lines.test.js

```diff
@@ -51,6 +51,40 @@ describe('Transform invoice lines', () => {
     })
   })
 
+  test('transforms CS invoice line', async () => {
+    const lineData = ['L', 'CS000000001', '100', '2023', '5704A', 'ERD14', 'A01000000001/MT', 'NE00', 'Y', '1', 'G00 - Gross value of claim', '2023-12-01', 'SOS273']
+    const result = transformInvoiceLine(lineData, cs.schemeId)
+    expect(result).toEqual({
+      invoiceNumber: 'CS000000001',
+      value: 100,
+      marketingYear: 2023,
+      schemeCode: '5704A',
+      fundCode: 'ERD14',
+      agreementNumber: 'A01000000001/MT',
+      deliveryBody: 'NE00',
+      convergence: true,
+      description: 'G00 - Gross value of claim',
+      dueDate: '2023-12-01',
+      accountCode: 'SOS273'
+    })
+  })
+
+  test('transforms BPS invoice line', async () => {
```

Comment:
This probably a nitpick, but could you move the bps test above cs? Just so it's consistent with the order of this 'const { sfi, sfiPilot, lumpSums, bps, cs } = require('../../../../app/schemes')'

Reply:
agree, changes made.